packages/frontend/app/services/new-version.js

```diff
@@ -0,0 +1,57 @@
+import Service from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import Ember from 'ember';
+import { task, timeout } from 'ember-concurrency';
+import config from 'frontend/config/environment';
+
+const ONE_MINUTE = 60000;
+
+export default class NewVersionService extends Service {
+  @tracked latestVersion = undefined;
+
+  get currentVersion() {
+    return config.newVersion.currentVersion;
+  }
+
+  get isNewVersionAvailable() {
+    return this.latestVersion && this.currentVersion !== this.latestVersion;
+  }
+
+  get url() {
```

Comment:
This should go into the [fetch service](https://github.com/ilios/frontend/blob/master/packages/ilios-common/addon/services/fetch.js) where most of this is done already and is a nice single place to look for networking stuff.

Reply:
so i stripped the implementation of this `url()` getter down a bit. please see https://github.com/ilios/frontend/pull/7770/commits/147359c9b6138d36200dbbd9b101403aaa179233

re: using our own `fetch` service for this - i had a look and it's built for backend API interactions, and not for anything else.

Reply:
Makes sense. ✅